### PR TITLE
add linux package dependency installation instructions to docs

### PR DESCRIPTION
**Pull request by kyleparrott**

### CODE REVIEW

**Comment by rplauche** [comment]
Looks good to me

File: docs/kubos-sdk.md [hunk 1]
```diff
@@ -35,6 +35,20 @@ Using homebrew:
 
 		$ brew install pip
 
+## Installing Dependencies
+
+#### Linux
+##### Ubuntu/Debian
+
+        $ sudo apt-get install build-essential libxml2-dev libxslt1-dev zlib1g-dev  wget python-dev  libffi-dev libssl-dev python-setuptools
```
Comment (CatFreed) [inline docs/kubos-sdk.md]: I also had to install libjim-dev and libdw-dev in order to get 'kubos flash' to run successfully

Comment (kyleparrott) [inline docs/kubos-sdk.md]: Good catch. Thanks for the reminder!

**Comment by rplauche** [comment]
Looks good

**Comment by kyleparrott** [comment]
Sweet, Merging.

**Pull request merged**